pkg/rpc/adminservice/execution.go

```diff
@@ -66,6 +66,31 @@ func (m *AdminService) RelaunchExecution(
 	return response, nil
 }
 
+func (m *AdminService) RecoverExecution(
+	ctx context.Context, request *admin.ExecutionRecoverRequest) (*admin.ExecutionCreateResponse, error) {
+	defer m.interceptPanic(ctx, request)
+	requestedAt := time.Now()
+	if request == nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Incorrect request, nil requests not allowed")
+	}
+	var response *admin.ExecutionCreateResponse
+	var err error
+	m.Metrics.executionEndpointMetrics.relaunch.Time(func() {
```

Comment:
relaunch? should we have an additional metric for this?

Reply:
done

Reply:
its still relaunch?

Reply:
done for real, thanks